OsmAnd/res/layout/bottom_sheet_item_btn_with_icon_and_text.xml

```diff
@@ -31,7 +31,7 @@
 		android:paddingStart="@dimen/content_padding"
 		android:paddingEnd="@dimen/content_padding">
 
-		<net.osmand.plus.widgets.TextViewEx
+		<TextView
```

Comment:
Wrong replacement. osmand:typeface="@string/font_roboto_medium" might not work in this case.
Do not change UI classes if there no reason for that.